src/shogun/CMakeLists.txt

```diff
@@ -400,18 +403,22 @@ ELSE()
   SHOGUN_DEPENDENCIES(
     LIBRARY NLopt
     SCOPE PRIVATE
-    CONFIG_FLAG HAVE_NLOPT)
+    CONFIG_FLAG HAVE_NLOPT
+    GPL_ONLY)
```

Comment:
because nlopt is LGPL? note it's lgpl and not gpl

Reply:
yep you are right this we can link against

Reply:
I just realised we can also move back the `CNLOPTMinimizer` class to the main repo, which is very good as then all the GP codes are still in the BSD release!